pkgs/applications/misc/haxor-news/default.nix

```diff
@@ -1,20 +1,35 @@
-{ stdenv, python, fetchpatch }:
-
-with python.pkgs;
+{ stdenv, fetchFromGitHub, python3, fetchpatch }:
+
+
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      self = py;
+
+      # not compatible with prompt_toolkit >=2.0
+      prompt_toolkit = super.prompt_toolkit.overridePythonAttrs (oldAttrs: rec {
+        name = "${oldAttrs.pname}-${version}";
+        version = "1.0.18";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "09h1153wgr5x2ny7ds0w2m81n3bb9j8hjb8sjfnrg506r01clkyx";
+        };
+      });
+    };
+  };
+in
+with py.pkgs;
 
 buildPythonApplication rec {
   pname = "haxor-news";
-  version = "0.4.3";
-
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "5b9af8338a0f8b95a8133b66ef106553823813ac171c0aefa3f3f2dbeb4d7f88";
-  };
-
-  # allow newer click version
-  patches = fetchpatch {
-    url = "${meta.homepage}/commit/5b0d3ef1775756ca15b6d83fba1fb751846b5427.patch";
-    sha256 = "1551knh2f7yarqzcpip16ijmbx8kzdna8cihxlxx49ww55f5sg67";
+  version = "unstable-2020-10-20";
```

Comment:
I'm not sure here... I mean, does it hurt to keep the old release and have an `haxor-news-unstable` attribute?

I also think that this piece of software is nothing critical, so we _could_ do it this way, but I'd prefer another maintainer if we do it. You're welcome to add yourself (in a dedicated commit I guess) to the maintainers!

Reply:
if we do this, the old version will be marked vulnerable very soon, see #101964